src/Microsoft.Extensions.Configuration.AzureAppConfiguration/ConfigurationClientManager.cs

```diff
@@ -37,15 +36,16 @@ internal class ConfigurationClientManager : IConfigurationClientManager
         private readonly ConfigurationClientOptions _clientOptions;
         private readonly bool _replicaDiscoveryEnabled;
         private readonly SrvLookupClient _srvLookupClient;
+        private readonly string _validDomain;
 
         private IList<ConfigurationClientWrapper> _dynamicClients;
-        private IEnumerable<string> _validDomains;
         private DateTimeOffset _lastFallbackClientRefresh = default;
         private DateTimeOffset _lastFallbackClientRefreshAttempt = default;
         private Logger _logger = new Logger();
 
         private static readonly TimeSpan FallbackClientRefreshExpireInterval = TimeSpan.FromHours(1);
         private static readonly TimeSpan MinimalClientRefreshInterval = TimeSpan.FromSeconds(30);
+        private static readonly string[] TrustedDomainLabels = new[] { "azconfig", "azconfig-test", "appconfig", "appconfig-test" };

```

Comment:
We need to avoid azconfig-test and appconfig-test. We don't use that domain across all clouds.

I know it's to get PPE to work, but we need to solve the root problem which is the use of a domain other than "azconfig".